src/modules/auth/auth.controller.ts

```diff
@@ -36,15 +36,14 @@ export class AuthController {
       await user.createEmailConfirmationToken(user);
 
     try {
-      await this.emailService.sendVerificationEmail(
+      this.rabbitMQService.sendEmailConfiramtion(
```

Comment:
Overall, it's better to have more abstract names for your services. Name of the service shouldn't say anything about the implementation. You should be able to change implementation without changing the name. So I would probably have `this.emailService.sendVerificationEmail` that uses RabbitMQ inside.